unfoldingWord/translationCore
# DMG compression for builds on macOS

#### This pull request addresses:

This improves https://github.com/unfoldingWord-dev/translationCore/issues/2807 by allowing compressed DMG's to be built on macOS.
This also include some updates to the travis config that simply stub out some features needed if we end up automating this dmg creation on travis' macOS images.

#### How to test this pull request:

Check out this code on a mac and run:

```
npm i 
gulp build --osx
gulp release --osx
```
The result should be a `dmg` file with a size around 90MB.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3397)
